python/dgl/heterograph.py

```diff
@@ -68,7 +68,7 @@ def __init__(self,
             raise DGLError('The input is already a DGLGraph. No need to create it again.')
         if not isinstance(gidx, heterograph_index.HeteroGraphIndex):
             dgl_warning('Recommend creating graphs by `dgl.graph(data)`'
-                        ' instead of `dgl.DGLGraph(data)`.')
```

Comment:
This is intended because DGLHeteroGraph and DGLGraph are the same. We actually have an alias in the root package. We will definitely change the class name in the future.